src/main.rs

```diff
@@ -87,6 +88,9 @@ impl Resources {
         let turtleshell = load_texture("assets/Whale/TurtleShell(32x32).png").await?;
         turtleshell.set_filter(FilterMode::Nearest);
 
+        let shoes = load_texture("assets/Whale/Shoes(32x32).png").await?;
```

Comment:
Isnt item's texture enough?

Reply:
For reference, check https://github.com/fishfight/FishFight/blob/main/src/items/sword.rs#L50